### PR TITLE
Integration test for Is_Zero

### PR DESCRIPTION
**Pull request by Sh0g0-1758**
Fixed a bug in hintcode and added integration test.

### CODE REVIEW

File: pkg/hintrunner/zero/zerohint_ec.go [hunk 1]
```diff
@@ -736,7 +736,7 @@ func createEcMulInnerHinter(resolver hintReferenceResolver) (hinter.Hinter, erro
 // i.e, 1 if `x == 0`, 0 otherwise
 func newIsZeroNondetHint() hinter.Hinter {
 	return &GenericZeroHinter{
-		Name: "IsZeroConditional",
+		Name: "IsZeroNondet",
 		Op: func(vm *VM.VirtualMachine, ctx *hinter.HintRunnerContext) error {
 			//> x == 0
```
Comment (TAdev0) [inline pkg/hintrunner/zero/zerohint_ec.go:741]: can you also update the comment here, to have both hintcode (python one and real one in json file)

Comment (Sh0g0-1758) [reply pkg/hintrunner/zero/zerohint_ec.go]: done.

**Review by har777 — APPROVED** [review]

File: integration_tests/cairo_zero_hint_tests/is_zero.small.cairo [hunk 2]
```diff
@@ -0,0 +1,31 @@
+// Returns 1 if x == 0 (mod secp256k1_prime), and 0 otherwise.
+// Serves as integration test for the following hints : 
+// isZeroNondetCode
+// isZeroPackCode
+// isZeroDivModCode
+
+%builtins range_check
+
+from starkware.cairo.common.cairo_secp.field import is_zero, SumBigInt3
```
Comment (har777) [inline integration_tests/cairo_zero_hint_tests/is_zero.small.cairo:9]: try with `from starkware.cairo.common.cairo_secp.bigint3 import BigInt3`

Comment (Sh0g0-1758) [reply integration_tests/cairo_zero_hint_tests/is_zero.small.cairo:9]: what do you mean ? is_zero takes a `SumBigInt3` not `BigInt3` . And I tried doing `from starkware.cairo.common.cairo_secp.bigint3 import SumBigInt3` but it gave the same error. It works for me locally but it fails on the github CI.

Comment (TAdev0) [reply integration_tests/cairo_zero_hint_tests/is_zero.small.cairo:9]: agree with hari, the rust vm does this in cairo files :

```
from starkware.cairo.common.cairo_secp.bigint3 import BigInt3, SumBigInt3
```
maybe try importing both `BigInt3`  and `SumBigInt3` this way?

**Review by TAdev0 — DISMISSED** [review]

File: integration_tests/cairo_zero_hint_tests/is_zero.small.cairo (hunk 2 above)
Comment (Sh0g0-1758) [reply integration_tests/cairo_zero_hint_tests/is_zero.small.cairo]: Nope, it still doesn't work.

Comment (Sh0g0-1758) [reply integration_tests/cairo_zero_hint_tests/is_zero.small.cairo:9]: Bumped cairo-lang version.

**Pull request merged**